iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/VoltageLevelNodeBreakerViewSwitchAdderAdapter.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.SwitchKind;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class VoltageLevelNodeBreakerViewSwitchAdderAdapter extends AbstractIdentifiableAdderAdapter<VoltageLevel.NodeBreakerView.SwitchAdder> implements VoltageLevel.NodeBreakerView.SwitchAdder {
```

Comment:
Same remark about this class, and maybe the Voltage Level views

Reply:
Ok